resources/metadata/cnbBuild.yaml

```diff
@@ -258,12 +258,19 @@ spec:
             param: custom/buildSettingsInfo
       - name: createBOM
         type: bool
-        description: "**EXPERIMENTAL:** Creates the bill of materials (BOM) using CycloneDX plugin."
+        description: Creates the bill of materials (BOM) using Syft and stores in a file in CycloneDX 1.4 format.
         scope:
           - GENERAL
           - STEPS
           - STAGES
           - PARAMETERS
+      - name: syftDownloadUrl
```

Comment:
Why is it a config parameter? It probably makes snse to do it the same way as `kaniko`, but is there any benefit?

Reply:
Afaik the plan is to integrate it with renovate (if not already)